include/eve/module/core/regular/impl/abs.hpp

```diff
@@ -36,12 +36,18 @@ abs_(EVE_SUPPORTS(cpu_), T const& a) noexcept
 
 // -----------------------------------------------------------------------------------------------
 // Masked case
-template<conditional_expr C, value U>
+template<conditional_expr C, scalar_value U>
 EVE_FORCEINLINE auto
 abs_(EVE_SUPPORTS(cpu_), C const& cond, U const& t) noexcept
 {
   return mask_op(cond, eve::abs, t);
 }
+template<conditional_expr C, scalar_value U, typename N>
+EVE_FORCEINLINE auto
+abs_(EVE_SUPPORTS(cpu_), C const& cond, wide<U,N> const& t) noexcept
```

Comment:
How did cpu work without it

Reply:
At one point, I got bad dispatch and it looked like splitting scalar + simd checks helped.
Now, I have to verify this isn't a leftover from earlier tests. It may be not required.

Reply:
Please try to remove